testing/simapp/simd/main.go

```diff
@@ -11,9 +11,8 @@ import (
 )
 
 func main() {
-	rootCmd, _ := cmd.NewRootCmd()
-
-	if err := svrcmd.Execute(rootCmd, "simd", simapp.DefaultNodeHome); err != nil {
+	rootCmd := cmd.NewRootCmd()
+	if err := svrcmd.Execute(rootCmd, "", simapp.DefaultNodeHome); err != nil {
```

Comment:
The second arg is `envPrefix` which was previously "simd", does this make any difference? I assume its all g

Reply:
eh I suspect they're both wrong heh